about/changelog.md

```diff
@@ -8,6 +8,37 @@ keywords: [changelog, upgrades, updates, releases]
 
 All the latest features and updates to Timescale products.
 
+## 🛠️ Improved tooling in Timescale Cloud and new AI and Vector extension releases
+
+### 2FA Indicator
+We’ve added a 2FA status column to the Members page, allowing customers to easily see whether each project member has 2FA enabled or disabled.
+
+### CSV import tool in the Timescale console
+We have added a CSV import tool available on service creation within the Timescale console.  For all TimescaleDB services, after service creation you can: 
+* Choose a local file
+* Select the name of the data collection to be uploaded (default is file name)
+* Choose data types for each column
+* Upload the file as a new hypertable within your service
+Look for the `Import data from .csv` tile in the `Import your data` step of service creation.
+
+### CloudWatch Permissions via Role Assumption
+We've released permission granting via role assumption to CloudWatch. Role assumption is both more secure and more convenient for customers who no longer need to rotate credentials and update their exporter config.
+
+For more details take a look at [our documentation](https://docs.timescale.com/use-timescale/latest/metrics-logging/integrations/).
+
+### Replica lag
+Customers now have more visibility into the state of replicas running on Timescale Cloud. We’ve released a new parameter called Replica Lag within the Operations tab for both Read and High Availability Replicas. Replica lag is measure in bytes against the current state of the primary database. For questions or concerns about the relative lag state of your replica, please reach out to Customer Support.
```

Comment:
@ggodeke have you confirmed this is visible yet? Maybe only part of the upcoming HA changes?